pkg/aws/actuator/actutator_e2e_test.go

```diff
@@ -0,0 +1,16 @@
+//go:build e2e
+// +build e2e
+
+package actuator
+
+import (
+	"testing"
+)
+
+// Test_CheckSTS runs end-to-end tests to verify that the results of an STS workflow are successful.
+
+func Test_CheckSTS(t *testing.T) {
+	t.Run("test of STS", func(t *testing.T) {
```

Comment:
I hadn't considered that this test would run during normal unit tests so we should exclude this one in some way, perhaps like [this example in the Hive Makefile](https://github.com/openshift/hive/blob/928bef3a88c299ac96a5c8fd4eba4f1e92ccd2c1/Makefile#LL40C1-L41C53). I think we should also move this test into an e2e directory of some sort again [similarly to the Hive setup](https://github.com/openshift/hive/tree/master/test/e2e) to make it clear that this test isn't meant to be ran during unit testing.

Reply:
I take back the first part :) I see that we have[ that in CCO](https://github.com/openshift/cloud-credential-operator/blob/1261fa91b46943e3d46988bace39b295c09f7bef/Makefile#L46-L47). Maybe we should just move this test into an e2e directory.

Reply:
Okay, 2018c178 isolates the e2e tests along the hive model. I'm trying avoid fully following that, with the `hack/*.sh` and all because it seems unneeded so far. I left the e2e tagging, but maybe that's overkill with them moved to a new location. Lemme know what you think.

Reply:
That seems fine, thanks!